File: Programmers/Level1/나누어떨어지는숫자배열/나누어떨어지는숫자배열.py
def solution(arr, divisor):
    answer = []
    for num in arr:
        if not num % divisor:
            answer.append(num)
    if not answer:
        answer.append(-1)
    return sorted(answer)

arr = [2, 36, 1, 3]
divisor = 1

print(solution(arr, divisor))

File: Programmers/Level1/나누어떨어지는숫자배열/나누어떨어지는숫자배열.js
function solution(arr, divisor) {
  const sortedArr = arr.sort((a, b) => a - b);
  var answer = [];

  for (let i = 0; i < sortedArr.length; i++) {
    if (sortedArr[i] % divisor === 0) {
      answer.push(sortedArr[i]);
    }
  }

  return answer.length ? answer : [-1];
}

const arr = [5, 9, 7, 10];
const divisor = 5;

console.log(solution(arr, divisor));
